core/src/mindustry/input/InputHandler.java

```diff
@@ -241,11 +241,7 @@ public static void payloadDropped(Unit unit, float x, float y){
             pay.set(x, y);
             pay.dropLastPayload();
             pay.set(prevx, prevy);
-            pay.controlling().each(u -> {
-                if(u instanceof Payloadc p){
-                    p.dropLastPayload();
-                }
-            });
+            pay.controlling().each(u -> Call.payloadDropped(u, u.x, u.y));
```

Comment:
I know the `instanceof` check technically isn't necessary, but by omitting it you'll be sending extra payload drop packets to clients that won't do anything with it. It's best to keep the check there.

Reply:
Fixed. https://github.com/Anuken/Mindustry/pull/3010/commits/801eca5ca3b2ada72884880636c9344d6ea999ea